utoipa-gen/tests/path_response_derive_test.rs

```diff
@@ -58,7 +58,9 @@ test_fn! {
         (status = 200, description = "success"),
         (status = 401, description = "unauthorized"),
         (status = 404, description = "not found"),
-        (status = 500, description = "server error")
+        (status = 500, description = "server error"),
+        (status = "5XX", description = "all other server errors"),
+        (status = "default", description = "default")
```

Comment:
This is ok, but would be more beneficial if asserts were also added under line 83.

Reply:
thanks yep I wasn't entirely sure, how these tests worked, have added now